Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -216,10 +249,16 @@ public Task InvokeTextDocumentDidChangeAsync(LSP.DidChangeTextDocumentParams req
         public Task InvokeDidChangeConfigurationAsync(LSP.DidChangeConfigurationParams request)
             => _rpc == null ? Task.CompletedTask : _rpc.NotifyWithParameterObjectAsync("workspace/didChangeConfiguration", request);
```

Comment:
does _rpc ever equal null?

Reply:
It can if the attach hasn't occurred yet.